spec/views/hyrax/my/collections/index.html.erb_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe "hyrax/my/collections/index.html.erb", type: :view do
+  before do
+    assign(:managed_collection_count, 0)
+    resp = []
```

Comment:
I believe the preferred way to do this kind of stub is...
```
let(:resp) { double(docs: "", total_count: 11) }
```

This would replace...
```
resp = []
allow(resp).to receive(:docs).and_return("")
allow(resp).to receive(:total_count).and_return(11)
```